pkg/config/config_template.yaml

```diff
@@ -1073,6 +1100,33 @@ api_key:
     #         - "monitored:true"
     #         - "customKey:customValue"
 
+    ## @param ping - custom object - optional
+    ## Configure ICMP pings for all hosts in SNMP autodiscovery
+    ## Each device will be pinged with these settings each time the SNMP
+    ## check is run.
+    ##
+    ## By default, we try to use an unprivileged UDP socket to send ICMP
+    ## pings, but some linux systems require using a raw socket.
+    ## See details at the link below:
+    ## https://sturmflut.github.io/linux/ubuntu/2015/01/17/unprivileged-icmp-sockets-on-linux/
+    ##
+    ## If `linux.use_raw_socket` is set, you must enable the ping module
+    ## of the system-probe for elevated privileges. See
+    ## system-probe.yaml.example for details.
+    #
+    # ping:
+    #   enabled: true             # disabled by default
+    #   timeout: 3000             # timeout in milliseconds
+    #   count: 2                  # number of ping packets to send per check run
+    #   interval: 10              # time between sending pings (up to `count` packets) in milliseconds
+    #   linux:                    # linux specific configuration
+    #     use_raw_socket: true    # send pings in a privileged fashion using a raw socket
```

Comment:
Should this part be in `{{- if .PingModule }}` block?

Reply:
This part should not, there's two pieces potentially that a user would need to configure. The regular agent, and system-probe. The `PingModule` piece is configured to make sure that the system-probe config only shows up in the `system-probe.yaml.example`. The section highlighted here is part of the regular agent configuration.